### DISCUSSION TITLE
Retrieving as Template results in rename of CustomerKey

### DISCUSSION THREAD

**Discussion by jnguyenandersen** [Accenture/sfmc-devtools#196] [Q&A] [answered]
I've got a mature org that I'm looking to backup to a Sandbox Instance before rolling out a Sandbox to Production model. Also new to running a deployment process in general, so not sure if this is an Issue, or just a Q&A on how to run this properly.

I started a small test:
```mcdev rt <BU> dataExtension "<External Key of DE Here>" <market>```

When I take a look at the templated DE, the CustomerKey seems to have changed to be the same as the name of the DE. 
I actually don't want that to happen, it prevents me from doing an update to the same DE from another BU down the line because the CustomerKeys will be different. I had to manually type in the External Key I wanted it to be, and proceeded to build, and deploy based on that for any updates I run to deploy on the same data extension.

My question is- is there a way to manually override the automatic renaming of the CustomerKey?

**Comment by JoernBerkefeld** [comment]
Hi, sorry for the late reply. If you just want to copy things you could use r instead of rt? That would avoid the override as well

**Comment by JoernBerkefeld** [comment, marked as answer]
@jnguyenandersen we removed the renaming in a currently open PR after careful consideration. your request certainly made me reconsider the issue. stay tuned for release 4.0.0 or PR #301
